class.jetpack.php

```diff
@@ -568,28 +568,20 @@ static function delete_active_modules() {
 	public function add_configure_hook() {
 		global $wp_filter;
 
-		$current_priority = has_filter( 'plugins_loaded', array( $this, 'configure' ) );
-		if ( false !== $current_priority ) {
-			remove_action( 'plugins_loaded', array( $this, 'configure' ), $current_priority );
+		if ( false === has_filter( 'plugins_loaded', [ $this, 'configure' ] ) ) {
```

Comment:
While we can use short array syntax in PHP 5.6+, WordPress core opted against recommending their use in their standards. You can [find out more here](https://make.wordpress.org/core/2019/07/12/php-coding-standards-changes/).

We follow Core's lead in Jetpack, so do you think you could update your PR to use `array()`?

Thanks!

Reply:
Sure @jeherve :)